instructor/models.py

```diff
@@ -187,17 +199,17 @@ class FlowCytometrySamplePrep(models.Model):
 
 class FlowCytometryHistogram(models.Model):
     sample_prep = models.ForeignKey(FlowCytometrySamplePrep, related_name='histograms')
-    strain_protocol = models.ForeignKey(StrainProtocol, related_name='histograms')
+    strain_protocol = models.ForeignKey(StrainTreatment, related_name='histograms')
     kind = models.CharField(max_length=50, choices=HISTOGRAMS, default=GAUSS)
     data = models.TextField(null=True,blank=True)
     enabled = models.BooleanField(default=False)
 
 admin.site.register(Course)
 admin.site.register(Assignment)
-admin.site.register(Protocol)
+admin.site.register(Drug)
 admin.site.register(Strains)
-admin.site.register(StrainProtocol)
-admin.site.register(Treatments)
+admin.site.register(StrainTreatment)
+admin.site.register(Treatment)
 admin.site.register(WesternBlot)
 admin.site.register(WesternBlotAntibody)
 admin.site.register(WesternBlotAntibodyBands)
```

Comment:
These changes raises a huge red flag: are you renaming the models or re-creating them?

Looking at the code (and at the previous migration) it looks you are creating new ones and deleting the old ones. Are you sure you do not have any data you need to preserve?